src/main/java/Duke.java

```diff
@@ -1,10 +1,259 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
-    public static void main(String[] args) {
+    //List of Special User Commands
+    public static final String EXIT_STRING = "bye";
```

Comment:
Good use of using constants preventing the issue of magical numbers or in this case magical strings